frontend/utilities/permissions/permissions.ts

```diff
@@ -1,6 +1,10 @@
 import { IUser } from "interfaces/user";
 import { IConfig } from "interfaces/config";
 
+export const isSandboxMode = (config: IConfig): boolean => {
+  return !!config.sandbox_enabled; // TODO: confirm null/undefined config should treated as false based on final API spec
```

Comment:
Good call out. I'm hoping the infrastructure sets it to true, but Tomas mentioned that it's possible it may set the value to `undefined`, in which case we'd need to treat `undefined` as `true`.